Xamarin.PropertyEditing.Mac/Controls/CollectionEditorWindow.cs

```diff
@@ -26,19 +26,28 @@ public CollectionEditorWindow (IHostResourceProvider hostResources, CollectionPr
 
 			ContentView.AddSubview (this.collectionEditor);
 
-			this.ok = NSButton.CreateButton (Properties.Resources.OK, OnOked);
-			this.ok.AccessibilityEnabled = true;
-			this.ok.AccessibilityTitle = Properties.Resources.AccessibilityCollectionOKButton;
-			this.ok.Highlighted = true;
-			this.ok.TranslatesAutoresizingMaskIntoConstraints = false;
-			
-			//this.ok.KeyEquivalent = "\r"; // FIXME: The type selector popup doesn't eat this key, so it ends up closing both.
+			this.ok = new FocusableButton {
```

Comment:
These buttons are already focusable, what's the need to change it?

Reply:
They aren't though. They don't allow the user to tab to them in `master`.

Reply:
Why this ? https://github.com/xamarin/Xamarin.PropertyEditing/blob/f2f3549c656b566a32ee52538298f06ce43190d3/Xamarin.PropertyEditing.Mac/Controls/Custom/FocusableButton.cs#L8

A disabled view cannot be KeyView by default, no?